tests/test_my_projects.py

```diff
@@ -17,6 +17,7 @@ def my_projects_page(driver):
     return my_projects_page
 
 
+@pytest.mark.skip(reason='Skip this test until bug ticket ENG-1737 is resolved')
```

Comment:
Are prod, staging1, and staging3 affected by this?  If not, you might want to put in a condition as the first arg to only skip for test and staging2.  [`test_dashboard`](https://github.com/cos-qa/osf-selenium-tests/blob/develop/tests/test_dashboard.py#L69) has a test that does this.

If the intention is to disable on all environments, then disregard this comment and :+1: to merge.

Reply:
Re: prod runs - because this section of the test creates a project and creates a collection, i.e. creates new data, we don't run it on prod, it is already marked  `@markers.dont_run_on_prod`.

I did a little checking, and for all staging/test servers the My Projects page still gets briefly tested in [test_navbar.py](https://github.com/cos-qa/osf-selenium-tests/blob/develop/tests/test_navbar.py) with a link click and assert using the page identity. So we'd at least know the page was returning a 404 or 500 or something else glaring.

Reply:
To your point about staging1 and staging3, I am keen to keep testing those if it makes sense to do that.  We only test them "on demand" and not on a schedule.  But there's still some value in keeping this test active on staging1 and staging3...as long as it's not going to always fail.

Reply:
Very good point you guys!  

I ran`test_my_projects.py` locally and it appears to be unstable on stage1 and stage3. Our selenium test suite is geared towards test.osf.io at the moment so I'll leave the all environments disabled until we start working towards modifying our tests to be server-specific.